kolibri/utils/cli.py

```diff
@@ -274,13 +300,16 @@ def _setup_django(debug):
         raise
 
 
-def initialize(skip_update=False):
+def initialize(skip_update=False):  # noqa: max-complexity=12
     """
     Currently, always called before running commands. This may change in case
     commands that conflict with this behavior show up.
     """
+    setup_logging(debug=get_debug_param())
     params = get_initialize_params()
 
```

Comment:
```suggestion
    # Check if the current user is the kolibri user when running kolibri from Debian installer.
```

Reply:
This should be fixed... also seems Github's suggestion system isn't working that well, at least here it displays as if you are suggesting to insert an orphaned comment instead of a necessary function call :)